lib/index.js

```diff
@@ -132,7 +132,19 @@ internals.specials = function () {
     }
 
     for (let i = 0; i < specials.length; ++i) {
-        lookup[specials.charCodeAt(i)] = true;
+        lookup[specials.codePointAt(i)] = true;
+    }
+
+    // add C0 control characters
+
+    for (let i = 0; i < 33; ++i) {
```

Comment:
Since this package only supports node 4+, you can probably use [`lookup.fill`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/fill) to initialize it.